core/services/keeper/registry_interface.go

```diff
@@ -18,7 +19,8 @@ import (
 type RegistryVersion int32
 
 const (
-	RegistryVersion_1_0 RegistryVersion = iota
+	ActiveUpkeepIDBatchSize int64           = 10000
+	RegistryVersion_1_0     RegistryVersion = iota
```

Comment:
nit. can we have ActiveUpkeepIDBatchSize as a separate const? Let's keep the registryVersion as an isolated enum declaration.